packages/js/src/addon-installation-successful.js

```diff
@@ -0,0 +1,14 @@
+import { render } from "@wordpress/element";
+import AddonInstallationSuccessful from "./components/AddonInstallationSuccessful";
+
+const elementToInsert = document.createElement( "div" );
+elementToInsert.setAttribute( "id", "wpseo-app-element-2" );
```

Comment:
Could you make the ID a bit more descriptive?
Suggestion: `wpseo-app-addon-installation-success-message`